.github/workflows/backend_node_actions.yml

```diff
@@ -26,20 +26,24 @@ jobs:
     defaults:
       run:
         shell: bash
-        working-directory: ./backend
-        
+
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [13.x]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm i
-    - run: npm run build --if-present
-    - run: npm run test:unit
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: cd ./backend && npm ci
```

Comment:
This action file is backend specific, why remove working-direction?

Reply:
I've added back in working-directory.